components/brave_shields/browser/base_brave_shields_service.cc

```diff
@@ -41,28 +32,6 @@ bool BaseBraveShieldsService::IsInitialized() const {
   return initialized_;
 }
 
-void BaseBraveShieldsService::DownloadDATFile() {
```

Comment:
I think this will break httpsE and tracking protection, so you could do those too if you want in this PR.

Reply:
nvermindI see the changes in this PR.